boards/arm/tmo_dev_edge/tmo_dev_edge-map.yml

```diff
@@ -0,0 +1,10 @@
+- connected: true
```

Comment:
Ditto - this doesn't seem to be a standard Zephyr board-level file.

Reply:
It is required to do device level twister testing, so I'm really surprised to hear you say that, but you are right, it's only used by twister and we use it nightly for board testing.

Reply:
You're right that you can save twister cmdline arguments to a HW map file, but it's not something that should be commited.

If you want to include this information nonetheless, you should do it by adding it to the platform documentation file, like e.g. `intel_adsp_cavs25` does.

Reply:
In general - board directories tend to be very standardized in Zephyr, and several tools depend on this standardization of data. Your board directory should +/- resemble what is defined in the [Board Porting Guide](https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#create-your-board-directory).